tasks/localizationtasks.js

```diff
@@ -24,228 +26,142 @@ const iso639_3_to_2 = {
 	trk: 'tr'
 };
 
+const supportedLocLangs = [
```

Comment:
Can you add a comment with a link to the list of valid loc strings?

Reply:
I just got it from the existing loc folders in the repo. I don't know if we have some list anywhere.